packages/office-addin-test-server/src/testServer.ts

```diff
@@ -3,37 +3,40 @@
 
 import * as cors from "cors";
 import * as express from "express";
+import * as http from "http";
 import * as https from "https";
 import * as devCerts from "office-addin-dev-certs";
 
-export const defaultPort: number = 4201;
+export const defaultHttpPort: number = 4200;
+export const defaultHttpsPort: number = 4201;
 
 export class TestServer {
     private jsonData: any;
-    private port: number;
+    private httpPort: number;
+    private httpsPort: number;
     private testServerStarted: boolean;
     private app: express.Express;
     private resultsPromise: Promise<JSON>;
-    private server: https.Server;
+    private httpServer: http.Server;
+    private httpsServer: https.Server;
 
     constructor(port: number) {
```

Comment:
why can you change https port but not http port?

Reply:
This would be better as `constructor(httpsPort: number = defaults.httpsPort, httpPort: number = defaults.httpPort)`

This means that they automatically use the default unless explicitly overridden. If the defaults are OK, then you can just use `new TestServer()`

Reply:
fixed locally